conf/modules.config

```diff
@@ -30,7 +30,16 @@ process {
         ext.when        = { !params.no_read_QC }
     }
 
-    withName: CUSTOM_DUMPSOFTWAREVERSIONS {
+    withName: 'MULTIQC' {
+        ext.args   = { params.multiqc_title ? "--title \"$params.multiqc_title\"" : '' }
+        publishDir = [
+            path: { "${params.outdir}/multiqc" },
+            mode: params.publish_dir_mode,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        ]
+    }
+
+    withName: 'CUSTOM_DUMPSOFTWAREVERSIONS' {
```

Comment:
we shouldn't need this one

Reply:
Ok, I think it was in the template update, but I couldn't find any calls to `CUSTOM_DUMPSOFTWAREVERSIONS`. Since you say it isn't need, out it goes ...